cloud/resource_cloud_environment.go

```diff
@@ -90,7 +90,7 @@ func resourceCloudEnvironment() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				Description:  descriptions["region"],
-				ValidateFunc: validateRegion,
+				ValidateFunc: validateNotBlank,
```

Comment:
But it's useful for AWS and GCP, can we create separate rule for Azure?

Reply:
I've updated this PR to add separate validation rules: https://github.com/streamnative/terraform-provider-streamnative/pull/69/files#diff-c02472e6df317b435706b287c76b320eea0b27f53a2bc8c447058fcc2be79cf6R227